styles.css

```diff
@@ -0,0 +1,667 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@700&family=Roboto:ital,wght@0,100;1,100&display=swap");
```

Comment:
Great design @Developer-Kingz 🤝

- [ ] As stated in the project requirements, kindly ensure your design uses the `Cocogoose, Lato` font family across your design.